pkg/client/fake/client_test.go

```diff
@@ -235,7 +236,7 @@ var _ = Describe("Fake client", func() {
 					Namespace: "ns2",
 				}
 				obj := &corev1.ConfigMap{}
-				err = cl.Get(nil, namespacedName, obj)
```

Comment:
Why this change is required?

Reply:
When running the linter with staticcheck enabled (not enforced by this PR) it fails with

`SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use (staticcheck)`

https://staticcheck.io/docs/checks#SA1012